src/App.css

```diff
@@ -1,5 +1,6 @@
 .App {
   text-align: center;
```

Comment:
Please remove this so that you can left align below

Reply:
ok removed so the only thing on the CSS file should be : 
```css
.App {
  width: 100vw;
}
```